### PR TITLE
Pass rc version to e2e tests

### PR DESCRIPTION
**Pull request by nemni8**
enables to pass extraEditorUrlParams to corvid-e2e-tests in order to run against a specific RC version.

### CODE REVIEW

File: packages/corvid-e2e-tests/test/drivers/cliDriver.js [hunk 1]
```diff
@@ -1,30 +1,43 @@
...
-  const login = ({ remoteDebuggingPort } = {}) => {
-    const query = parseCommandArgs({ remoteDebuggingPort });
-    return execa.shell(`${CORVID_BIN_PATH} login ${query}`, { cwd });
+  const withCommand = async (
+    commandName,
+    { editorUrl, remoteDebuggingPort }
+  ) => {
+    const port = remoteDebuggingPort || (await findFreePort());
+    const query = parseCommandArgs({ remoteDebuggingPort: port });
+    const url = editorUrl ? editorUrl : "";
```
Comment (tombenezra) [inline packages/corvid-e2e-tests/test/drivers/cliDriver.js]: I'd add this param to `parseCommandArgs` as well

File: packages/corvid-e2e-tests/test/loginLogout.e2e.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  "https://editor.wix.com/html/editor/web/renderer/edit/bdce5f39-c471-4198-9778-c6c8118dbaa8?metaSiteId=d387aea5-55d2-47cf-a348-3d1fbe316245&editorSessionId=bf5f376b-7485-4995-a6c6-d7535d9c7853";
+
+describe("login / logout", () => {
+  let cliDriver;
+
+  beforeEach(async done => {
+    const cwd = tempy.directory();
+    cliDriver = corvidCliDriverCreator({ cwd });
+    await (await cliDriver.logout()).waitForCommandToEnd();
+    done();
```
Comment (tombenezra) [inline packages/corvid-e2e-tests/test/loginLogout.e2e.js]: is this necessary?

File: packages/corvid-e2e-tests/test/loginLogout.e2e.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  });
+
+  afterAll(async done => {
+    await (await cliDriver.logout()).waitForCommandToEnd();
+    done();
```
Comment (tombenezra) [inline packages/corvid-e2e-tests/test/loginLogout.e2e.js]: same here, what's the point of `waitForCommandToEnd`?

File: packages/corvid-e2e-tests/test/drivers/localEditorDriver.js [hunk 4]
```diff
@@ -8,10 +8,9 @@ const THEMED_TEXTS_SELECTOR =
 
 module.exports = page => {
   const waitForEditor = async () =>
```
Comment (tombenezra) [inline packages/corvid-e2e-tests/test/drivers/localEditorDriver.js]: remove async

File: packages/corvid-e2e-tests/test/loginLogout.e2e.js (hunk 2 above)
Comment (nemni8) [reply packages/corvid-e2e-tests/test/loginLogout.e2e.js]: the idea is that **every cli command** will return the **same** cliCommandResult object, which has 2 methods and a property. the methods are:  waiting for the command to end and kill the command process programmatically and the editor debug port as property.

**Pull request merged**